BitwardenShared/Core/Auth/Services/AuthService.swift

```diff
@@ -718,13 +718,13 @@ class DefaultAuthService: AuthService { // swiftlint:disable:this type_body_leng
     /// - Parameter response: The response received from the identity token request.
     /// - Returns: Whether the vault can be unlocked with a Key Connector key.
     ///
-    private func canUnlockWithKeyConnectorKey(_ response: IdentityTokenResponseModel) async throws -> Bool {
+    private func canUnlockWithKeyConnectorKey(_ response: IdentityTokenResponseModel) async throws -> (Bool, URL?) {
```

Comment:
⛏️ Update docs on what is returned.
🤔 Would it be simpler to just return the `URL?` and change a bit the name of the method? I mean `nil` would mean can't unlock with key connector and `not nil` would mean can unlock with key connector. However it may make it harder to follow on the code though, what do you think?

Reply:
Yeah, I did like the name of that method, which is why I ended up the tuple. But it is simpler to return an optional URL. I ended up changing it to `func keyConnectorUrlForUnlock(_:) -> URL?`, which I think still reads ok.